src/sentry/middleware/integrations/parsers/discord.py

```diff
@@ -32,19 +32,35 @@ class DiscordRequestParser(BaseRequestParser):
     def get_integration_from_request(self) -> Integration | None:
         if self.view_class in self.control_classes:
             params = unsign(self.match.kwargs.get("signed_params"))
-            return Integration.objects.filter(id=params.get("integration_id")).first()
+            integration_id = params.get("integration_id")
+
+            logger.info(
+                f"{self.provider}.get_integration_from_request.{self.view_class.__name__}",
+                extra={"path": self.request.path, "integration_id": integration_id},
+            )
+            return Integration.objects.filter(id=integration_id).first()
 
         if self.view_class == DiscordInteractionsEndpoint:
             drf_request: Request = DiscordInteractionsEndpoint().initialize_request(self.request)
             discord_request: DiscordRequest = self.view_class.discord_request_class(drf_request)
 
             self.discord_request = discord_request
 
+            logger.info(
+                f"{self.provider}.get_integration_from_request.discord_interactions_endpoint",
+                extra={"path": self.request.path, "guild_id": discord_request.guild_id},
```

Comment:
The guild id could be `self.discord_request.guild_id` since it was just assigned in the line above, but doesn't really matter.

Reply:
Ah. I copied `discord_request.guild_id` from the lines of code below this one.